test/ng/directive/ngOptionsSpec.js

```diff
@@ -2532,8 +2532,8 @@ describe('ngOptions', function() {
       expect(scope.value).toBe(false);
 
       scope.$apply('required = true');
-      expect(element).toBeValid();
```

Comment:
Imo, this is not right. required means that at least one option must be selected, it doesn't matter if the value is false. I'll look into why this test is failing.